home-manager/core.nix

```diff
@@ -17,11 +17,11 @@ in
       group = "core";
       extraGroups = [ "wheel" ];
       openssh.authorizedKeys.keys = cfg.authorizedKeys;
-      shell = pkgs.fish;
+      #shell = pkgs.fish;
```

Comment:
Instead of commenting fish out, I think we should instead keep the PR open and merge once the build problem is resolved on the upstream.

Reply:
Agreed, let's revert this later.